Tests/SwiftDocCTests/Test Resources/TestBundle-RenderIndex.json

```diff
@@ -739,6 +739,15 @@
         "title" : "SideProtocol",
         "type" : "protocol"
       },
+      {
```

Comment:
These items are added because the UncuratedClass is part of the SideKit navigation hierarchy now.